site/content/docs/browser-checks/_index.md

```diff
@@ -219,6 +220,43 @@ While Playwright and Puppeteer share many similarities, they have evolved at dif
 
 We recommend using Playwright Test if you are just starting out or [migrating from Puppeteer to Playwright using `puppeteer-to-playwright`](https://github.com/checkly/puppeteer-to-playwright).
 
+## Using other browsers
+{{< info >}}
+We strongly recommend using the default `chromium` browser for all your checks [as per the offcial Playwright documentation](https://playwright.dev/docs/browsers#google-chrome--microsoft-edge).
+{{< /info >}}
+
+If your application [has very specific requirements](https://playwright.dev/docs/browsers#when-to-use-google-chrome--microsoft-edge-and-when-not-to), 
+Checkly enables you to use `Google Chrome` with Playwright in runtimes `2023.02` and newer. 
```

Comment:
```suggestion
Checkly enables you to use `Google Chrome` with Playwright in runtime `2023.02`. 
```
Let's remove the `and newer`. When we will release a new runtime, we totally forget about this two words and we might not want to support it after.